saleor/csv/utils/export.py

```diff
@@ -89,6 +90,37 @@ def export_gift_cards(
     send_export_download_link_notification(export_file, "gift cards")
 
 
+def export_voucher_codes(
+    export_file: "ExportFile",
+    file_type: str,
+    voucher_id: Optional[int] = None,
+    ids: Optional[List[int]] = None,
+    delimiter: str = ",",
+):
+    file_name = get_filename("voucher_code", file_type)
+
+    qs = VoucherCode.objects.all()
+    if voucher_id:
+        qs = VoucherCode.objects.filter(voucher_id=voucher_id)
+    if ids:
+        qs = VoucherCode.objects.filter(id__in=ids)
+
+    export_fields = ["code"]
```

Comment:
Sorry if asking something obvious: 
Don't we want to provide more data here than only `code`?  Like `voucherId`, `usage` etc?
For example, I am generating the CSV file. See one suspicious code. I don't have voucher ID, I have only `code`. 
How can I easily find the voucher on the dashboard/graphql side? I might be wrong but I don't see any filter on `vouchers` query that would allow mi to find the voucher with the specific `code`.

Reply:
@korycins As an MVP we just decided to keep it simple and allow to export only codes. If it will be needed we will add filters later.